src/Makefile

```diff
@@ -49,17 +49,17 @@ mo-doc: $(SOURCE_ID)
 	@ln -fs $(MO_DOC_TARGET) mo-doc
 
 moc.js: $(SOURCE_ID)
-	dune build $(DUNE_OPTS) $(MOC_JS_TARGET)
+	dune build $(DUNE_OPTS) --profile=release $(MOC_JS_TARGET)
```

Comment:
shall we add a new var that has `--profile=release` and then we link that for the .js targets? Maybe something like `JS_TARGET_OPTS` or so.

Reply:
I don't think that is necessary. Nobody builds these targets locally.